gtdynamics.i

```diff
@@ -81,24 +81,17 @@ class ContactPointFactor : gtsam::NoiseModelFactor {
 };
 
 #include <gtdynamics/factors/TwistFactor.h>
-gtsam::NonlinearFactor TwistFactor(const gtsam::noiseModel::Base *cost_model,
+gtsam::NonlinearFactor* TwistFactor(const gtsam::noiseModel::Base *cost_model,
```

Comment:
We need a Python unit test verifying this works before this can be merged.

Reply:
Python tests added